udata_front/theme/gouvfr/assets/js/dsfr.js

```diff
@@ -6,3 +6,41 @@
 window.dsfr = {
     mode: 'manual',
 }
+
+/**
+ * This function returns the dsfr alignment to use for flex main axis alignment.
+ *
+ * @remarks
+ * It abstracts the fact that the dsfr name is made for flex-direction row and is odd for flex-direction column.
+ *
+ * @param {import("./types").AxisAlignment} alignment
+ */
+export function getMainAxisAlignment(alignment) {
```

Comment:
It's not needed in this PR right, but is used in publishing form?

Reply:
It's used in the ActionCard component to allow to choose the buttons alignement. 
Here the value isn't provided so it defaults to "start".

Reply:
In the publishing form, the `end` alignment is used instead.